segmentstore/storage/src/main/java/io/pravega/segmentstore/storage/chunklayer/ChunkedSegmentStorageConfig.java

```diff
@@ -100,13 +105,34 @@
     @Getter
     final private boolean appendEnabled;
 
+    /**
+     * Whether the lazy commit functionality is enabled or disabled.
+     */
+    @Getter
+    final private boolean lazyCommitEnabled;
+
+    /**
+     * Whether the inline defrag functionality is enabled or disabled.
+     */
+    @Getter
+    final private boolean inlineDefragEnabled;
+
+    /**
+     * Level of self check functionality enabled.
+     */
+    @Getter
+    final private int selfCheckLevel;
```

Comment:
What about an enum?

Reply:
We should at least document (here too) what these numbers mean.

Reply:
removed this config.
It is better to enable individual checks separately.